src/atom_finder/classifier/curly-braces.clj

```diff
@@ -0,0 +1,30 @@
+(in-ns 'atom-finder.classifier)
+(import '(org.eclipse.cdt.core.dom.ast IASTCompoundStatement IASTIfStatement IASTForStatement IASTWhileStatement))
+
+
+(defn curly-braces?
+  "Does this note represent curly brace?"
+  [node]
+  (instance? IASTCompoundStatement node))
+  
+
+(defn curly-braciable-statement?
+  "Does this AST node an statement that should be followed by curly braces?"
+  [node]
+  (let [statement #{;statement that should be followed by compound statements (curly braces)
+                    IASTIfStatement IASTForStatement IASTWhileStatement}]
```

Comment:
This list of "statements" doesn't look exhaustive. Is there a better way to do this than by explicit list?